dataland-frontend/tests/e2e/fixtures/sme/SmePreparedFixtures.ts

```diff
@@ -24,6 +25,8 @@ export function generateSmePreparedFixtures(): Array<FixtureData<SmeData>> {
 function manipulateFixtureForYear(input: FixtureData<SmeData>, year: string): FixtureData<SmeData> {
   input.companyInformation.companyName = "SME-year-" + year;
   input.reportingPeriod = year;
+  //There is a test that requires to sectors to be present at the same time, so that the modal appears
```

Comment:
Is this comment required?   We could give that information in the JS doc of this function or make it clear in the function name.

Reply:
changed name removed comment